www/pagetpl.php

```diff
@@ -113,7 +114,22 @@ function pageHeader($title, $focusCtl = false, $extraOnLoad = false,
                     <li class="<?= ($pagescript === 'showuser') ? 'page-active':''; ?>"><a id="topbar-profile" href="/showuser">Profile</a></li>
                     <li class="<?= ($pagescript === 'editprofile') ? 'page-active':''; ?>"><a id="topbar-edit" href="/editprofile">Settings</a></li>
                     <li class="<?= ($pagescript === 'personal') ? 'page-active':''; ?>"><a id="topbar-personal" href="/personal">My Activity</a></li>
-                    <li class="<?= ($pagescript === 'commentlog') ? 'page-active':''; ?>"><a id="topbar-inbox" href="/commentlog?mode=inbox">Inbox</a></li>
+                    <li class="<?= ($pagescript === 'commentlog') ? 'page-active':''; ?>"><a id="topbar-inbox" href="/commentlog?mode=inbox">Inbox
+                    <?php
+                    // check the inbox so we can display the number of new messages
+                    $inboxCnt = 0;
+                    $uid = checkPersistentLogin();
+                    if ($uid) {
+                        $db = dbConnect();
+                        $quid = mysql_real_escape_string($uid, $db);
+                        [$inbox, $inboxCnt] =
+                        queryComments($db, "inbox", $quid, "limit 0, 1", $caughtUpDate, false);
```

Comment:
This line isn't working right. I see this warning in the Docker console:

```
PHP Notice:  Undefined variable: caughtUpDate in /var/www/html/pagetpl.php on line 125
```

And, sure enough, the number flagged in the inbox is the count of all comments for all time; even if you go to the inbox and click "I'm caught up," the Inbox header number doesn't decrease or go away.

`home` sets `$caughtUpDate` on line 27. I think we'd need to do something similar here in `pagetpl.php`.

Reply:
Ok, $caughtUpDate is now set.